components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/actions/DashboardAction.kt

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package io.gitpod.jetbrains.remote.actions
+
+import com.intellij.openapi.actionSystem.AnAction
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.components.service
+import io.gitpod.jetbrains.remote.GitpodManager
+import org.apache.http.client.utils.URIBuilder
+
+class DashboardAction : AnAction() {
+    private val manager = service<GitpodManager>()
+
+    override fun actionPerformed(event: AnActionEvent) {
+        manager.pendingInfo.thenAccept { workspaceInfo ->
+            URIBuilder(workspaceInfo.gitpodHost).setPath("workspaces").build().toString().let { url ->
```

Comment:
```suggestion
            URIBuilder(workspaceInfo.gitpodHost).build().toString().let { url ->
```

For [consistency with VS Code Browser](https://github.com/gitpod-io/openvscode-server/blob/gp-code/main/extensions/gitpod-shared/src/features.ts#L336-L343) I think we should load the root url of the dashboard, which eventually redirects to `/workspaces`

Reply:
Agreed. I'll update it!